src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -183,6 +183,7 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
           endDate: moment()
             .add(395, 'days')
             .format('YYYY-MM-DD'),
+          useV2: featureVAOSServiceRequests,
```

Comment:
I was hoping to the use VAOS service appointments flag in this ticket: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/redux/selectors.js#L66

Reply:
Done.